File: php/Ram.php
<?php


class Ram
{
    private $capacity, $price;


    public function __construct($capacity = 0, $price = 0)
    {
        $this->capacity = $capacity;
        $this->price = $price;
    }

    public function setCapacity($capacity)
    {
        $this->capacity = $capacity;
    }

    public function setPrice($price)
    {
        $this->price = $price;
    }

    public function getCapacity()
    {
        return $this->capacity;
    }

    public function getPrice()
    {
        return $this->price;
    }
}

File: python/Ram.py
class Ram():

    def __init__(self, capacity=0, price=0):
        self.__capacity = capacity
        self.__price = price

    def setCapacity(self, capacity):
        self.__capacity = capacity

    def setPrice(self, price):
        self.__price = price

    def getCapacity(self):
        return self.__capacity

    def getPrice(self):
        return self.__price
